tests/Resources/ti.ui.toolbar.test.js

```diff
@@ -0,0 +1,219 @@
+				title: 'Item 1',
+				showAsAction: Ti.Android.SHOW_AS_ACTION_NEVER
+			});
+		};
+		window.addEventListener("open", function() {
+			should(toolbar.contentInsetEndWithActions).eql(20);
+		});
+		window.open();
+	});
+
+	(utilities.isAndroid() ? it : it.skip)('contentInsetStartWithNavigation', function() {
+		var toolbar = Ti.UI.createToolbar({contentInsetStartWithNavigation: 20, top: 0, width: Ti.UI.FILL});
+		window.activity.supportToolbar = toolbar;
+		window.add(toolbar);
+		window.addEventListener("open", function() {
+			window.activity.actionBar.displayHomeAsUp = true;
+			should(toolbar.contentInsetStartWithNavigation).eql(20);
+		});
+		window.open();
+	});
+
+	it('extendBackground', function() {
+		var toolbar = Ti.UI.createToolbar({contentInsetStartWithNavigation: 20, top: 0, width: Ti.UI.FILL, extendBackground: true, barColor: 'red'});
+		window.activity.supportToolbar = toolbar;
+		window.add(toolbar);
+		window.addEventListener("open", function() {
+			should(toolbar.extendBackground).eql(true);
+		});
+		window.open();
+	});
+
+	(utilities.isAndroid() ? it : it.skip)('logo', function(finish) {
+		var toolbar = Ti.UI.createToolbar({top: 0, width: Ti.UI.FILL, logo: Ti.Filesystem.resourcesDirectory + 'Logo.png', barColor: 'blue'});
+		window.activity.supportToolbar = toolbar;
+		window.add(toolbar);
+		toolbar.addEventListener('resourceLoaded', function(e) {
+			should(e.target).eql('logo');
+			should(e.loaded).eql(true);
+			finish();
+		});
+		window.open();
+	});
+
+	(utilities.isAndroid() ? it : it.skip)('logo pixel density specific', function(finish) {
+		var toolbar = Ti.UI.createToolbar({top: 0, width: Ti.UI.FILL, logo: '/images/dip.png', barColor: 'blue'});
+		window.activity.supportToolbar = toolbar;
+		window.add(toolbar);
+		toolbar.addEventListener('resourceLoaded', function(e) {
+			should(e.target).eql('logo');
+			should(e.loaded).eql(true);
+			finish();
+		});
+		window.open();
+	});
+
+	(utilities.isAndroid() ? it : it.skip)('navigationIcon', function(finish) {
+		var toolbar = Ti.UI.createToolbar({top: 0, width: Ti.UI.FILL, navigationIcon: Ti.Filesystem.resourcesDirectory + 'Logo.png', barColor: 'blue'});
+		toolbar.addEventListener('resourceLoaded', function(e) {
+			should(e.target).eql('navigationIcon');
+			should(e.loaded).eql(true);
+			finish();
+		});
+		window.activity.supportToolbar = toolbar;
+		window.add(toolbar);
+		window.open();
+	});
+
+	(utilities.isAndroid() ? it : it.skip)('navigationIcon pixel density specific', function(finish) {
+		var toolbar = Ti.UI.createToolbar({top: 0, width: Ti.UI.FILL, barColor: 'blue', navigationIcon: '/images/dip.png'});
+		toolbar.addEventListener('resourceLoaded', function(e) {
+			should(e.target).eql('navigationIcon');
+			should(e.loaded).eql(true);
+			finish();
+		});
+		window.activity.supportToolbar = toolbar;
+		window.add(toolbar);
+		window.open();
+	});
+
+	(utilities.isAndroid() ? it : it.skip)('overflowIcon', function(finish) {
```

Comment:
If the test is intended to be android specific (it's an API meant to only be on android and isn't an API that's broken/missing on other platforms), just do:
`it.android('overflowIcon', function (finish) {`